web/js/waste.js

```diff
@@ -92,6 +92,8 @@ $(function() {
         var valueAttribute = $this.find('option').filter(':selected').data('extra');
         valueAttribute = valueAttribute ? valueAttribute.message : '';
         if (valueAttribute) {
+            // Make sure utf8 characters display correctly
+            valueAttribute = decodeURIComponent(escape(valueAttribute));
```

Comment:
escape is not valid modern JavaScript - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/escape - what is this line attempting to do/fix, what was the issue with utf8 characters?

Reply:
Iirc, on the item selection page, utf8 chars (like ☃, as an extreme case) were showing up as garbled, despite looking fine elsewhere.

Reply:
Hmm, the issue needs sorting at source, then, because we shouldn't require a line like this, everything should be encoded correctly already.
![image](https://user-images.githubusercontent.com/154364/213154434-daf1f36e-46aa-47ba-9570-7f52075c6ebb.png)
I can take a look.

Reply:
Ah, thank you 👍